core/dbt/events/functions.py

```diff
@@ -133,7 +140,7 @@ def cleanup_event_logger():
 EVENT_MANAGER.add_logger(
     _get_logbook_log_config(debug=False)  # type: ignore
     if ENABLE_LEGACY_LOGGER
-    else _get_stdout_config(log_format="text", debug=False, use_colors=True)  # type: ignore
+    else _get_stdout_config(log_format="text", debug=False, use_colors=True, log_cache_events=True, quiet=False)  # type: ignore
```

Comment:
I think we'll want `log_cache_events=False` by default

FYI @peterallenwebb since you're:
- assigned to #6979
- thinking about how to handle `--quiet` in the default logger, before we have full config (#6847) — it's just being set to `False` here

To be honest, I don't know if it's possible, and it might be motivation for us to really finally uncouple `UserConfig` from `profiles.yml` (#6207), so that we can have a configured logger before we need to parse the profile (and encounter any warnings)

Reply:
Updated, what exactly does it do?

Reply:
Super verbose logging every time a relation is added, removed, updated from the cache. Adds a ton of noise in the general case, but very useful to turn on when debugging issues that might be due to cache misses/inconsistencies